src/ImageSequencer.js

```diff
@@ -212,7 +212,10 @@ ImageSequencer = function ImageSequencer(options) {
     let inputs = copy(modulesInfo(step.options.name).inputs);
     inputs = inputs || {};
 
-    for(let input in inputs) inputs[input] = step.options[input] || inputs[input].default;
+    for(let input in inputs) {
+      inputs[input] = step.options[input] || inputs[input].default;
+      inputs[input] = inputs[input].toString().replace(',',',%20');
```

Comment:
Try this -- `encodeURIComponent()` -- this is available in node too. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent